qiskit/pulse/library/waveform.py

```diff
@@ -26,11 +26,14 @@ class Waveform(Pulse):
     duration of the backend cycle-time, dt.
     """
 
+    limit_amplitude = True
```

Comment:
This will only impact the `Waveform` pulse. I wonder if this global setting should live with `Pulse` which will allow these checks to be disabled for parametric pulses as well?

Reply:
@taalexander I would be happy to move the attribute to the `Pulse` class and make the checks in the parametric pulses depend in this (this was one of the options mentioned in #6012) Can you discuss internally and let me know whether we go this way?

Reply:
I am fine with it 😄, I believe it will lead to the most expected behaviour.